<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>mainWindow</class>
 <widget class="QMainWindow" name="mainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1579</width>
    <height>840</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>反原斗士</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label_img">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>100</y>
      <width>564</width>
      <height>360</height>
     </rect>
    </property>
    <property name="text">
     <string>原图</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_img_ostu">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>251</width>
      <height>201</height>
     </rect>
    </property>
    <property name="text">
     <string>大津法</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_img_adp">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>480</y>
      <width>251</width>
      <height>201</height>
     </rect>
    </property>
    <property name="text">
     <string>自适应</string>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="plainTextEdit_0">
    <property name="geometry">
     <rect>
      <x>1180</x>
      <y>60</y>
      <width>301</width>
      <height>241</height>
     </rect>
    </property>
    <property name="documentTitle">
     <string notr="true"/>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1579</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>V1.0</string>
    </property>
   </widget>
   <addaction name="menu"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
